src/views/webview-app/components/connect-form/connection-form.tsx

```diff
@@ -33,27 +33,6 @@ type DispatchProps = {
 type props = StateProps & DispatchProps;
 
 export class ConnectionForm extends React.Component<props> {
-  /**
-   * Renders a component with messages.
-   *
-   * @returns {React.Component}
-   */
-  renderConnectionMessage(): React.ReactNode {
-    const { isConnected, connectionMessage } = this.props;
-
-    if (isConnected && connectionMessage) {
-      return (
-        <div className={styles['connection-message-container']}>
-          <div className={styles['connection-message-container-success']}>
-            <div
-              className={styles['connection-message']}
-            >{connectionMessage}</div>
-          </div>
-        </div>
-      );
-    }
-  }
```

Comment:
This was unused. Would probably be good to have some stricter typescript rules 🤔 although it probably wouldn't catch cases like this since it's a function in an exported class.